sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_receipts_async.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_receipts_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze receipts from a file.
+
+USAGE:
+    python sample_recognize_receipts_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+"""
+
+import os
+import asyncio
+from pathlib import Path
+
+
+class RecognizeReceiptsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    async def recognize_receipts(self):
+        # the sample forms are located in this file's parent's parent's files.
+        path_to_sample_forms = Path(__file__).parent.parent.absolute() / Path("sample_forms/receipt/contoso-allinone.jpg")
+        # TODO: this can be used as examples in sphinx
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+        async with FormRecognizerClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        ) as form_recognizer_client:
+
+            with open(path_to_sample_forms, "rb") as f:
+                receipts = await form_recognizer_client.recognize_receipts(stream=f.read())
```

Comment:
@kristapratico we don't need to do `asUSReceipt` for the following field values?

Reply:
Python will differ in the design here - we will return a Union[USReceipt, UKReceipt, etc] instead of calling asUSReceipt on a RecognizedReceipt

Reply:
How do you decide whether a recognized receipt is a US receipt?

Reply:
right now, US is the only supported type of receipt. In the future, the `receipt_locale` will be used.

Reply:
> In the future, the `receipt_locale` will be used.

It is going to come from the service?  Sounds reasonable.  I will update JS to do the same.

Reply:
yes, as a param on the receipt method.